### PR TITLE
Use proper link for log/build view

### PR DESCRIPTION
**Pull request by quvide**

### CODE REVIEW

File: client/src/build-view.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Build extends React.Component {
 
   render() {
     const { name } = this.props.match.params
-    const url = `http://${name}.localhost:8080`
+    const url = `http://${name}.${window.location.host}`
```
Comment (bqqbarbhg) [inline client/src/build-view.js:48]: Is there any way using the dynamic browser location could fail?
There's `meta/env.meta.js` that should contain a statically configured URL based on build-time environment variables.

Comment (quvide) [reply client/src/build-view.js:48]: It's done this way in `container-card.js` too. The url in `env.meta.js` has the protocol too as part of it, would it be better then just to parse the hostname out of that?

**Comment by HenrikHelenius** [comment]
@quvide @bqqbarbhg i have nothing to add

File: client/src/build-view.js (hunk 1 above)
Comment (HenrikHelenius) [reply client/src/build-view.js:48]: @quvide do you need @bqqbarbhg 's input?

Comment (bqqbarbhg) [reply client/src/build-view.js:48]: I suppose it's fine like that, you could add another meta constant that doesn't have the protocol though

**Review by bqqbarbhg — APPROVED** [review]
I guess it's fine

**Pull request merged**